server/package.json

```diff
@@ -18,6 +18,7 @@
     "express": "^4.18.3",
     "jsonwebtoken": "^9.0.2",
     "mongoose": "^8.2.2",
+    "multer": "^1.4.5-lts.1",
```

Comment:
What is multer?

Reply:
Multer is a node.js middleware for handling multipart/form-data, which is primarily used for uploading files. It is written on top of [busboy](https://github.com/mscdex/busboy) for maximum efficiency.